src/Uno.UI/UI/Xaml/Shapes/ArbitraryShapeBase.wasm.cs

```diff
@@ -144,6 +144,12 @@ private Rect GetBBoxOfChildrenWithStrokeThickness()
 
 			foreach (FrameworkElement child in GetChildren())
 			{
+				if (child.HtmlTag == "linearGradient")
```

Comment:
You should put this in a `<defs>` tag an just filter-out this one.

Reply:
done